### PR TITLE
Fix drop deconv

### PR DESCRIPTION
**Pull request by ceci3**
replace conv2d_transpose link
replace conv3d_transpose link
increase description of the output in conv2d_transpose
fix mistake in dropout

### CODE REVIEW

**Comment by ceci3** [comment]
conv2d_transpose增加output_size的解释如下
屏幕快照 2019-07-26 上午10 36 29

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -3728,8 +3728,15 @@ def conv2d_transpose(input,
...
            H^\prime_{out} &= (H_{in} - 1) * strides[0] - 2 * paddings[0] + dilations[0] * (H_f - 1) + 1 \\\\
            W^\prime_{out} &= (W_{in} - 1) * strides[1] - 2 * paddings[1] + dilations[1] * (W_f - 1) + 1 \\\\
-           H_{out} &\in [ H^\prime_{out}, H^\prime_{out} + strides[0] ) \\\\
-           W_{out} &\in [ W^\prime_{out}, W^\prime_{out} + strides[1] )
+           H_{out} &\in [ H^\prime_{out}, H^\prime_{out} + strides[0] ] \\\\
+           W_{out} &\in [ W^\prime_{out}, W^\prime_{out} + strides[1] ] 
+
+    Note:
+          if output_size is None, :math:`H_{out} == H^\prime_{out}, W_{out} == W^\prime_{out}`; 
+          else, the :math:`H_{out}` of the output size must between in :math:`H^\prime_{out}`
```
Comment (heavengate) [inline python/paddle/fluid/layers/nn.py]: between in -> between

**Review by wanghaoshuang — DISMISSED** [review]
LGTM

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -3728,8 +3728,15 @@ def conv2d_transpose(input,
 
            H^\prime_{out} &= (H_{in} - 1) * strides[0] - 2 * paddings[0] + dilations[0] * (H_f - 1) + 1 \\\\
            W^\prime_{out} &= (W_{in} - 1) * strides[1] - 2 * paddings[1] + dilations[1] * (W_f - 1) + 1 \\\\
-           H_{out} &\in [ H^\prime_{out}, H^\prime_{out} + strides[0] ) \\\\
-           W_{out} &\in [ W^\prime_{out}, W^\prime_{out} + strides[1] )
+           H_{out} &\in [ H^\prime_{out}, H^\prime_{out} + strides[0] ] \\\\
+           W_{out} &\in [ W^\prime_{out}, W^\prime_{out} + strides[1] ] 
+
+    Note:
+          if output_size is None, :math:`H_{out} == H^\prime_{out}, W_{out} == W^\prime_{out}`;
```
Comment (heavengate) [inline python/paddle/fluid/layers/nn.py]: 这里应该是==还是=？

Comment (ceci3) [reply python/paddle/fluid/layers/nn.py]: 已修改，谢谢~

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (ceci3) [reply python/paddle/fluid/layers/nn.py]: 已修改，谢谢~

**Review by heavengate — APPROVED** [review]
LGTM

**Review by guoshengCS — APPROVED** [review]

**Pull request merged**